tests/syslog_health_monitor_test.go

```diff
@@ -185,6 +179,170 @@ func TestSyslogHealthMonitorXIDDetection(t *testing.T) {
 	testEnv.Test(t, feature.Feature())
 }
 
+// TestSyslogHealthMonitorXIDFloodWithDuplicates tests burst XID injection
+func TestSyslogHealthMonitorXIDFloodWithDuplicates(t *testing.T) {
```

Comment:
Can we merge this into the truncation test cases? So we have a ton of XID like how we have in the read world, we can check that the first few XIDs show up correctly and then the rest are truncated, what do you think?